manual/histograms/index.md

```diff
@@ -88,30 +87,31 @@ For all histogram types: `nbins`, `xlow`, `xup`:
 
   - The last bin (bin# `nbins+1`) contains the overflow.
 
-  - In case of 2-D or 3-D histograms, a *global bin* number is defined.
-
-_**Example**_
+  - A *global bin* number is defined  to access the histogram bin information independently of the dimension. 
 
-Assuming a 3-D histogram `h` with `binx`, `biny`, `binz`, the function returns a global/linear bin number.
+Assuming a 3-D histogram `h` with `binx`, `biny`, `binz`, the function [TH1::GetBin(binx,bny,binz)](https://root.cern/doc/master/classTH1.html#a641262682144d465d7e2bc6101a04bf6) returns a global
+bin number and given a global bin number `bin`, the function [TH1::GetBinXYZ(bin,binx,biny,binz)](https://root.cern/doc/master/classTH1.html#a55e591270aaad37c3059a62f83566e4e) computes the
+corresponding `binx`, `biny` and `binz`.
 
-`Int_t bin = h->GetBin(binx, biny, binz);`
+More details on the histogram binning are available in the TH1 [reference documentation](https://root.cern/doc/master/classTH1.html#binning).
+ROOT supports also automatic binning, see [Histograms with automatic bins](https://root.cern/doc/master/classTH1.html#auto-bin).
 
-This *global bin* is useful to access the bin information independently of the dimension.
 
 **Re-binning**
 
 You can re-bin a histogram via the [TH1::Rebin()](https://root.cern/doc/master/classTH1.html#aff6520fdae026334bf34fa1800946790){:target="_blank"} method. It returns a new histogram with the re-binned contents. If bin errors were stored, they are recomputed during the re-binning.
+You can see this [tutorial](https://root.cern/doc/master/rebin_8C.html) as a re-binning example.
 
 ### Stack of histograms
 
-  - {% include ref class="THStack" %} is a collection of {% include ref class="TH1" %} or {% include ref class="TH2" %} histograms.
+{% include ref class="THStack" %} is a collection of {% include ref class="TH1" %} or {% include ref class="TH2" %} histograms. The tutorial [hstack.C](https://root.cern/doc/master/hstack_8C.html) is a good example of how using the `THStack` class.
```

Comment:
You can use: `{% include tutorial name="hstack" %}`to point to `hstack.C`

Reply:
Thank you , I will update the links